funding/manager.go

```diff
@@ -3670,6 +3735,13 @@ func (f *Manager) announceChannel(localIDKey, remoteIDKey *btcec.PublicKey,
 		return err
 	}
 
+	// After the fee parameters have been stored in the announcement
+	// we can delete them from the database.
+	err = f.deleteInitialFwdingPolicy(chanID)
```

Comment:
In the pr review club we noticed that this is only called when the channel is announced. If we create a private channel with custom feerates, this will never be called and the forwarding policy would never be deleted.